portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,144 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
+const COMMENT_SIZE_LIMIT = 100;
+
 function getRandomFact() {
-  const greetings =
-      ['My ex podmate stole my go link', 'I was the shortest person in my junior high', 'I know 3 languages', 'C++ is my favorite programming language'];
+  const facts =
+    ["My ex podmate stole my go link", "I was the shortest person in my junior high", "I know 3 languages", "C++ is my favorite programming language"];
+
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
+}
+
+const createCommentElement = commentPayload => {
+  console.log(commentPayload);
+  const commentElement = document.createElement("li");
```

Comment:
I'm not the most proficient web dev, but it feels like this is a lot of HTML building in JavaScript, which makes it hard to style the UI easily (you'd have to understand all the JS just to move a div). Ideally, you want to shift as much of the styling burden to your HTML/CSS as possible.

Consider putting a template comment element in HTML body of your .html file, and then using JS to read the HTML element and manipulate the contents. Something like https://www.w3schools.com/jsref/met_node_clonenode.asp . There might be better ways to do something similar, that was just the first Google result I got.

That way it's easy to go to the html file, mess with the styling, and not worry about the JS.

Reply:
Done. I still had to specify every element, but it is definitely cleaner